documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -48,6 +48,11 @@ on GitHub.
   by an executable standalone JAR distribution. For details, see the "New Features" section below.
 * The `MethodSortOrder` enum in `ReflectionUtils` has been renamed to `HierarchyTraversalMode`. Those
   affected should now use `ReflectionSupport` instead of `ReflectionUtils`.
+* The method `execute(LauncherDiscoveryRequest launcherDiscoveryRequest)` in `Launcher` was
+  deprecated and will be removed in milestone M5. Instead use new method that registers supplied
+  `TestExecutionListener`s in addition to already registered listeners but only for the
```

Comment:
This formatting doesn't work AsciiDoc. See my recent commit addressing that issue. 😉 

Basically, you have to wrap the _code_ in double back-ticks if you place any character directly adjacent to the formatted code: \`\`TestExecutionListener\`\`s